src/layout/FileUpload/FileUploadTable/FileTableComponent.module.css

```diff
@@ -13,6 +13,10 @@
   border-spacing: 0;
 }
 
+.summaryTableMargin {
+  margin-top: 0.6rem;
+}
+
```

Comment:
Denne marginen ser ut til å overstyre marginen som blir satt i css klassene `table` og `tableMobile` uavhengig av om tabellen er i mobilvisning eller ikke. Kunne vi kanskje heller bare endret margin-top verdien som befinner seg der? 

Hva kommer verdien 0,6rem av? Det ser ut til å rendres som 10px. Helst så bør vi prøve å bruke designsystemet sine spacing tokens. Der har vi desverre ikke en token for 10px, men 8 og 12. (`var(--fds-spacing-2)` = 8px, `var(--fds-spacing-3)` = 12px).

Reply:
Godt poeng! Den marginen som er satt i hovedklassen er ganske massiv, og det fungerer greit nok når man har det "drop area" over tabellen for å dra filer inn. Men ser ganske fjernt ut hvis man er på maks antall attachments, og drop area forsvinner. Det ser også ganske fjernt ut i summary, så jeg la inn en override.

`0.6rem` var vel bare litt øyemål og bør sikkert fikses. Målet var å ende opp med ca samme spacing som default komponentene i Summary2 (see screenshot i issue linket til PR).

Målet mitt med hele PRen var vel bare å fikse noen åpenbare mangler, ikke ta på meg å endre hele komponenten. Så jeg forsøkte å bare endre det lille jeg ville ha på plass for at Summary2 skulle se bedre ut.

Jeg kan ta en runde til på mobil vs normal, og bruke en av variablene som margin 👌